### PR TITLE
[Core] Cleaning python exposition

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

Clean up header. Pragma once. Unify namespace.

**🆕 Changelog**

- [Clean up header. Pragma once](https://github.com/KratosMultiphysics/Kratos/commit/cf1ab0458242c68db43b2dba04f4252937118a9c)
- [More clean up (unify namespace)](https://github.com/KratosMultiphysics/Kratos/commit/48a1092d7b10a2d461e08654bbd21e70049cbbeb)

### CODE REVIEW

File: kratos/python/add_amgcl_solver_to_python.h [hunk 1]
```diff
@@ -4,18 +4,14 @@
 //   _|\_\_|  \__,_|\__|\___/ ____/
 //                   Multi-Physics
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
 //  Main authors:    Riccardo Rossi
+//                   Pooyand Dadvand
```
Comment (rubenzorrilla) [inline kratos/python/add_amgcl_solver_to_python.h]: Be aware of the authorship changes...

File: kratos/python/add_cfd_variables_to_python.h [hunk 2]
```diff
@@ -4,39 +4,25 @@
 //   _|\_\_|  \__,_|\__|\___/ ____/
 //                   Multi-Physics
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
-//  Main authors:    Pooyan Dadvand
-//                   Riccardo Rossi
```
Comment (rubenzorrilla) [inline kratos/python/add_cfd_variables_to_python.h]: Same comment. I'd leave the original author order.

File: kratos/python/add_constitutive_law_to_python.h [hunk 3]
```diff
@@ -4,39 +4,25 @@
...
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
-//  Main authors:    Pooyan Dadvand
-//                   Riccardo Rossi
+//  Main authors:    Riccardo Rossi
+//                   Pooyand Dadvand
```
Comment (rubenzorrilla) [inline kratos/python/add_constitutive_law_to_python.h]: Same.

File: kratos/python/add_containers_to_python.h [hunk 4]
```diff
@@ -4,39 +4,25 @@
...
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
-//  Main authors:    Pooyan Dadvand
-//                   Riccardo Rossi
+//  Main authors:    Riccardo Rossi
+//                   Pooyand Dadvand
```
Comment (rubenzorrilla) [inline kratos/python/add_containers_to_python.h]: Same.

File: kratos/python/add_dem_variables_to_python.h [hunk 5]
```diff
@@ -4,40 +4,25 @@
...
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
-//  Main authors:    Pooyan Dadvand
-//                   Riccardo Rossi
+//  Main authors:    Riccardo Rossi
+//                   Pooyand Dadvand
```
Comment (rubenzorrilla) [inline kratos/python/add_dem_variables_to_python.h]: Same.

File: kratos/python/add_deprecated_variables_to_python.h [hunk 6]
```diff
@@ -4,39 +4,25 @@
...
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
-//  Main authors:    Pooyan Dadvand
-//                   Riccardo Rossi
+//  Main authors:    Riccardo Rossi
+//                   Pooyand Dadvand
```
Comment (rubenzorrilla) [inline kratos/python/add_deprecated_variables_to_python.h]: Same.

File: kratos/python/add_dofs_to_python.h [hunk 7]
```diff
@@ -8,32 +8,21 @@
 //                   Kratos default license: kratos/license.txt
 //
 //  Main authors:    Riccardo Rossi
+//                   Pooyand Dadvand
```
Comment (rubenzorrilla) [inline kratos/python/add_dofs_to_python.h]: Same.

File: kratos/python/add_geometrical_utilities_to_python.h [hunk 8]
```diff
@@ -7,34 +7,22 @@
 //  License:         BSD License
 //                   Kratos default license: kratos/license.txt
 //
-//  Main authors:    Pooyan Dadvand
-//                   Riccardo Rossi
+//  Main authors:    Riccardo Rossi
```
Comment (rubenzorrilla) [inline kratos/python/add_geometrical_utilities_to_python.h]: Same.

File: kratos/python/add_geometries_to_python.h [hunk 9]
```diff
@@ -1,39 +1,28 @@
...
-//                   Multi-Physics 
+//                   Multi-Physics
 //
-//  License:		 BSD License 
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
 //  Main authors:    Riccardo Rossi
+//                   Pooyand Dadvand
```
Comment (rubenzorrilla) [inline kratos/python/add_geometries_to_python.h]: Same.

File: kratos/python/add_global_pointers_to_python.h [hunk 10]
```diff
@@ -4,36 +4,25 @@
 //   _|\_\_|  \__,_|\__|\___/ ____/
 //                   Multi-Physics
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
 //  Main authors:    Riccardo Rossi
+//                   Pooyand Dadvand
```
Comment (rubenzorrilla) [inline kratos/python/add_global_pointers_to_python.h]: Same.

File: kratos/python/add_io_to_python.h [hunk 11]
```diff
@@ -4,39 +4,25 @@
...
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
-//  Main authors:    Pooyan Dadvand
-//                   Riccardo Rossi
+//  Main authors:    Riccardo Rossi
+//                   Pooyand Dadvand
```
Comment (rubenzorrilla) [inline kratos/python/add_io_to_python.h]: Same.

File: kratos/python/add_cfd_variables_to_python.h (hunk 2 above)
Comment (loumalouomega) [reply kratos/python/add_cfd_variables_to_python.h]: (Okay, for me the copy pasting was very practical)

File: kratos/python/add_kernel_to_python.h [hunk 12]
```diff
@@ -4,38 +4,25 @@
...
 //
-//  License:		 BSD License
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
-//  Main authors:    Pooyan Dadvand
-//                   Riccardo Rossi
+//  Main authors:    Riccardo Rossi
+//                   Pooyand Dadvand
```
Comment (rubenzorrilla) [inline kratos/python/add_kernel_to_python.h]: Same.

File: kratos/python/add_model_to_python.h [hunk 13]
```diff
@@ -2,39 +2,27 @@
...
+//                   Multi-Physics
 //
-//  License:		 BSD License 
-//					 Kratos default license: kratos/license.txt
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
 //
 //  Main authors:    Riccardo Rossi
-//                   Pooyan Dadvand
+//                   Pooyand Dadvand
```
Comment (rubenzorrilla) [inline kratos/python/add_model_to_python.h]: Typo.

**Review by rubenzorrilla — CHANGES_REQUESTED** [review]
Clean up looks good to me. However there are some changes in the authorship (names and order) that I'd leave as they used to be. Besides you introduced a typo in Pooyand's name.

**Comment by loumalouomega** [comment]
> Clean up looks good to me. However there are some changes in the authorship (names and order) that I'd leave as they used to be. Besides you introduced a typo in Pooyand's name.

Done, funny thing is that the typo comes from a copy paste from one header...

**Comment by loumalouomega** [comment]
Done @rubenzorrilla

**Review by rubenzorrilla — APPROVED** [review]
Very good. Thanks for the effort.

**Pull request merged**